lib/Tga/SimHash/Comparator/GaussianComparator.php

```diff
@@ -36,6 +36,28 @@ public function __construct($deviation = 4)
     }
 
     /**
+     * Count Differences beetwin fingerprints.
+     *
+     * @param int $size
+     * @param Fingerprint $fp1
+     * @param Fingerprint $fp2
+     * @return int
+     */
+    private static function countDifferences(int $size, Fingerprint $fp1, Fingerprint $fp2) {
```

Comment:
I'm pretty sure this method is slower than the current one, why do you need this?

Reply:
Because in the current implementation of 128bit version, there is a loss of precision in the conversion.

Reply:
Got it, thanks :) .

Reply:
You can't use scalar type hint, we support php 5.3+

Reply:
remove  scalar type hint